### ISSUE TITLE
An ADA issue: the control's «reset» state is not announced by the screen reader when user selects the «Reset» button on the «Store Locator» page

### ISSUE THREAD

**Issue by dmitry-fedyuk** [kitchencollection-com/core#30]
[upwork.com/ab/f/contracts/20874289](https://www.upwork.com/ab/f/contracts/20874289)
[kitchencollection.com/storelocator](https://www.kitchencollection.com/storelocator)

> Reset alert should be announced by the screen reader this will make the user get a clear concise
that any filter is applied or not, else a user with visual impairment who relies on screen reader for
web access will need to navigate the page manually using arrow keys in order to check for alerts.

**Comment by dmitry-fedyuk** [comment]
Fixed in [0.2.0](https://github.com/kitchencollection-com/core/releases/tag/0.2.0).

**Comment by dmitry-fedyuk** [comment]
![06](https://user-images.githubusercontent.com/83204/47508144-599d2e00-d895-11e8-9047-fb44131b5704.png)
